package-versions/packageversion.md

```diff
@@ -1,105 +1,95 @@
-
-#### Directory.Build.props
-```xml
-<Project>
-  <PropertyGroup>
-    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
-  </PropertyGroup>
-</Project>
-```
-
-To date, our engineering efforts for CPVM have been focused on supporting restore operations for development and CI/CD pipelines. In the future, we are planning to add additional tooling for Visual Studio.
-
-## Runtime Targeting with .NET 5 SDK
-Updates to CPVM will only be included in the .NET 5 SDK. You can target .NET Core and .NET Framework from the .NET 5 SDK.
-
-.NET Core runtimes and SDK's are decoupled, but they share the same major and minor versions. Any version of the .NET Core or .NET 5 SDK can build any .NET Framework app or any .NET Core app that is equal to or lower than the version of the SDK installed if the relevant Targeting Pack is installed.
-
-Currently, using the .NET 5 SDK with Visual Studio requires installing the preview version of Visual Studio. In the future when .NET 5 is released, it will be possible to install the .NET 5 SDK in the General Availability channel of Visual Studio version 16.8.
-
-## What works today?
-- Visual Studio 16.7 command line restore operations
-- Visual Studio GUI restore operations
-
-## What doesn't work today?
-- Adding a package via the NuGet package management interface in Visual Studio does not currently support CPVM. Once a package has been added using a GUI, it is necessary to add a corresponding <PackageVersion> entry within Directory.Packages.props.
-- Similarly, modifying a package version using the Visual Studio package management interface will update the project file, and it is necessary to move this Version="x.x.x" attribute to the corresponding <PackageVersion> entry within Directory.Packages.props.
+---
+layout: default
+title: Package Versions
+nav_order: 2
+has_children: true
+---
+
+# Central Package Version Management
+Central Package Version Management allows every project within a solution to source their NuGet package versions from a single `Directory.Packages.props` file at the solution level.
+
+## Managing Package Versions at the Solution Level
+
+Managing dependencies across many projects, solutions, and repositories can be very difficult at scale as `<PackageReference>` can be defined in project files, *.props files, and *.targets files throughout the code.
+
+By centrally managing package versions, solutions can maintain more consistency in their NuGet dependencies across projects.
+
+## Transitive Dependencies
+
+If any packages are imported as transitive dependencies from a `<PackageReference>` where the `Version` is controlled by a `<PackageVersion>` in `Directory.Packages.props`, then the version of the transitive dependency will be controlled by CPVM.
+
+If a `<PackageVersion>` has been defined for the package that is imported transitively, then that version will be imported across the solution.
```

Comment:
How is this different from the paragraph above it? It sounds like the same thing to me.

Reply:
This is a good point.

Created Issue #3 

Pushed a fix